reference_data/platform_flavors.yaml

```diff
@@ -291,3 +291,34 @@
       production: false
       debug: false
       priority: 10
+
+- name: openqa
+  repositories:
+    - name: remi
```

Comment:
Shouldn't this be named differently? i.e.: openqa-almalinux-9

Reply:
Changed it, thank you)